src/Screens/DiveLogsForms/components/ImagePickerArray.tsx

```diff
@@ -153,27 +160,34 @@ const ImagePickerArray: FunctionComponent<ImagePickerArrayProps> = ({
           </ScrollView>
         </View>
       ) : (
-        <View style={styles.subContainer}>
-          <Pressable onPress={openCameraModal}>
+        <Pressable
+          onPress={openCameraModal}
+          style={({ pressed }) => [
+            {
+              transform: pressed ? [{ scale: 0.95 }] : [{ scale: 1.0 }],
```

Comment:
checkout this effect. kinda cool makes the button feel very responsive

Reply:
That's pretty nice. Good job!